packages/node/src/index.ts

```diff
@@ -18,22 +18,23 @@ export { LogtoError, OidcError, Prompt, LogtoRequestError, LogtoClientError } fr
 
 export default class LogtoClient extends BaseClient {
   constructor(config: LogtoConfig, adapter: Pick<ClientAdapter, 'navigate' | 'storage'>) {
-    const { appId, appSecret } = config;
-
     super(config, {
       ...adapter,
       requester: createRequester(
-        appSecret
+        config.appSecret
```

Comment:
wrong base?

Reply:
There's a new lint rule that makes sure the `super` comes first in the constructor. Therefore we can't declare variables before the `super` call. Have to make this change to adopt the new rule.